src/python/library/tritonclient/utils/__init__.py

```diff
@@ -151,6 +151,9 @@ def np_to_triton_dtype(np_dtype):
         return "FP64"
     elif np_dtype == np.object_ or np_dtype.type == np.bytes_:
         return "BYTES"
+    elif (str(np_dtype) == "<class 'bfloat16'>"):
+        if np_dtype == bfloat16:
```

Comment:
need the import here too right?

Reply:
Also are we leaving it as an optional dependency user has to install then? If so we should document that somewhere noticeable.

Reply:
If the dtype is already there, it means the user has already imported bfloat16 right? The user does not have to install bfloat16, it is already in the sdk container. I can add a comment here if they are doing their own thing in their container. 

They will need to import bfloat16, which I assume they will do because they are converting a dtype to triton type and therefore already using it somewhere else

Reply:
> The user does not have to install bfloat16, it is already in the sdk container.

if they're using the client outside of the SDK container, they will. such as on jetson where we install wheel directly, right?

>If the dtype is already there, it means the user has already imported bfloat16 right?

Your logic sounds reasonable that if they passed the type, it must already be imported - but I suspect a couple potential issues:
- If they are using a different bf16 library that does the same thing (patching a numpy type), there may be a potential issue?
- Also if for some reason they did `import bfloat16 as bf16` in their code or used a differently named package, I believe this `if np_dtype == bfloat16` would fail to resolve the name/package.

Reply:
1. Added comments to address your concern with regards to user needing to install bfloat16
2. With regards to your concern about importing after the type->string check:
   a. It is true. I think importing bfloat16 as we expect it might not save us
   b. string-ifying classes will give you the actual type name. Eg:
   
`>>> from bfloat16 import bfloat16 as bf
>>> str(bf)
"<class 'bfloat16'>"

`